### ISSUE TITLE
Double-click to enable/disable scrolling for output

### ISSUE THREAD

**Issue by twopwood** [jupyterlab/jupyterlab#11625]
Relating to issue [#5897](https://github.com/jupyterlab/jupyterlab/issues/5897) one thought would be to allow double-clicking on the blue bar to enable/disable scrolling for output.  In fact, since there is currently no behavior assigned to double-clicking, perhaps this could even be a customizable feature where any of the options from right-clicking could be assigned as double-clicking behavior.

**Comment by jtpio** [comment]
Thanks @twopwood for opening this.

Since https://github.com/jupyterlab/jupyterlab/issues/5897 sounds very similar, maybe we should only keep one issue and move the conversation there? Or do you think the "double-click" aspect should be tracked separately?

**Comment by twopwood** [comment]
Yes, it makes sense to combine with #5897.  I just wanted to make sure the "double-click" feature gets considered since I think it could give more ways to interact without overriding any existing behaviors.  Thanks for considering this.

**Comment by MMesch** [comment]
Since #5897 was closed this issue stands now independently for the requested double click feature request which is not implemented in JLab 4.4.3.
